pyscada/hmi/static/pyscada/js/pyscada/pyscada.js

```diff
@@ -611,7 +613,7 @@ function timeline_drag( event, ui ) {
     update_timeline();
 }
 function PyScadaPlot(id){
```

Comment:
maybe fork the whole PyScadaPlot function and make a dedicated PyScadaPlotXY  and PyScadaPlotTimeY function.